pkgs/data/icons/capitaine-cursors-themed/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, stdenvNoCC
+, fetchzip
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "capitaine-cursors-themed";
+  version = "5";
+  src = fetchzip {
```

Comment:
I remember the people like to use fetchzip directly, instead of calling mkDerivation.

You can look for examples in `data/fonts`.

Reply:
I think it's done, can you review it again?